src/runtime/threading_backend.cc

```diff
@@ -0,0 +1,120 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file threading_backend.cc
+ * \brief Native threading backend
+ */
+#include <tvm/runtime/threading_backend.h>
+#include <dmlc/logging.h>
+#include <thread>
+#if defined(__linux__)
+#include <sched.h>
+#endif
+
+namespace {
+int MaxConcurrency() {
```

Comment:
let us still expose MaxConcurrency as function, because this will allow it to be lazily called as opposed to eagerly initialized via static variable